cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/remote/RemoteExecutionTwillController.java

```diff
@@ -115,23 +116,27 @@ public void release() {
     executionService.stop();
   }
 
-  public void complete() {
+  public void complete () {
     terminateOnServiceStop = true;
     executionService.stop();
-
     try {
+      RuntimeJobStatus status;
       RetryStrategy retryStrategy = RetryStrategies.timeLimit(
         5, TimeUnit.SECONDS, RetryStrategies.exponentialDelay(500, 2000, TimeUnit.MILLISECONDS));
 
       // Make sure the remote execution is completed
       // Give 5 seconds for the remote process to shutdown. After 5 seconds, issues a kill.
       long startTime = System.currentTimeMillis();
-      while (Retries.callWithRetries(remoteProcessController::isRunning, retryStrategy, Exception.class::isInstance)) {
+      while ((status = Retries.callWithRetries(
```

Comment:
indentation is off which makes this difficult to read, as one statement is broken up over 3 lines. This should be 2 lines

```
  while ((status = Retries.callWithRetries(
    remoteProcessController::isRunning, retryStrategy, Exception.class::isInstance)) == RuntimeJobStatus.RUNNING) {

  }
```